website/addons/mendeley/static/node-cfg.js

```diff
@@ -0,0 +1,101 @@
+var $ = require('jquery');
+var ko = require('knockout');
+var $osf = require('osfHelpers');
+require('./node-cfg.css');
+
+var MendeleyAccount = function(display_name, id) {
+    var self=this;
+    self.display_name = display_name;
+    self.id = id;
+};
+
+var CitationList = function(name, provider_list_id, provider_account_id ) {
+    var self=this;
+    self.name = name;
+    self.provider_list_id = provider_list_id;
+    self.provider_account_id = provider_account_id;
+};
+
+var MendeleySettingsViewModel = function() {
+    var self=this;
+
+    self.settings_url = nodeApiUrl + 'mendeley/settings/';
+
+    self.accounts = ko.observableArray();
+    self.selectedAccountId = ko.observable();
+    self.citationLists = ko.observableArray();
+    self.selectedCitationList = ko.observable();
+    self.message = ko.observable();
+
+    self.updateAccounts = function() {
+        $.getJSON(nodeApiUrl + 'mendeley/accounts/', function(data) {
+            for(var i=0; i<data.accounts.length; i++) {
+                self.accounts.push(new MendeleyAccount(
+                    data.accounts[i].display_name,
+                    data.accounts[i].id
+                ));
+            }
+            self.selectedAccountId(self.accounts()[0].id);
+            self.updateCitationLists();
+        }).fail(function() {
+            console.log("Failed to load list of accounts");
```

Comment:
Should the user see anything when this happens? Also, prefer to keep `console.log` statements out of production code.